107/src/main/java/org/ehcache/jsr107/ConfigurationMerger.java

```diff
@@ -162,6 +164,43 @@
     return builder;
   }
 
+  private static <K, V> CacheConfigurationBuilder<K, V> addDefaultCopiers(CacheConfigurationBuilder<K, V> builder, Class keyType, Class valueType ) {
```

Comment:
But in ehcache default copier is always 'IdentityCopier', no matter what is the type. So if no copier is provided IdentityCopier will be used. But in JSR, default copier is SerializingCopier.